### DISCUSSION TITLE
Vertical averaging/interpolation

### DISCUSSION THREAD

**Discussion by ecomodeller** [DHI/modelskill#17] [Ideas]
Should observations be:
1. compared to the model value in a given grid cell without interpolation
2. compared to model values interpolated to the depth of the observations
3. observed values averaged to match model grid

This is a common source of discussion in assessment of biogeochemical models.

Example of (hypothetical) observed data from some sensor along with model on a discrete non-equidistant vertical grid
![image](https://user-images.githubusercontent.com/614215/108121011-6b2c2480-70a2-11eb-81ff-776be0b845c7.png)

The same observations, now averaged for each model grid cell.
![image](https://user-images.githubusercontent.com/614215/108121047-767f5000-70a2-11eb-87d8-832cf5dde407.png)

**Comment by dhithu** [comment]
I guess this relates to how we do in the temporal domain, where (I think) we normally interpolate the model result to fit the observation. I guess, because the observation is considered "a data point of truth" and also because the model normally has a higher temporal resolution than the observation. For a vertical profile I guess both observation and model result could have the higher resolution (although models become increasingly finer), but none-the-less I would say that your item 2 would be the most intuitive way to go.
